### PR TITLE
[xcm-emulator] Unignore cumulus integration tests

### PR DESCRIPTION
**Pull request by gilescope**
The tests send the message sent on ininitalisation so for now adding in an empty block stops the tests from being ignored. (if the CI is happy then I will add in the other tests)

Also some other tests were falling when using `runtime-benchmarks` feature. Added `#[cfg(not(feature = "runtime-benchmarks"))]` to prevent them from being run.

### CODE REVIEW

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable 2/3
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/3474427

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable 2/3
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/3488270

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable 2/3
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/3502918

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable 2/3
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/3509756

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: check-tracing
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/3527671

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-clippy
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/3527989

**Review by NachoPal — COMMENTED** [review]
Can you add `#[cfg(not(feature = "runtime-benchmarks"))]` in
```
#[cfg(test)]
mod tests;
```
instead of in each particular test? We do not want to run any test when `runtime-benchmarks`

File: cumulus/parachains/integration-tests/emulated/assets/asset-hub-polkadot/Cargo.toml [hunk 1]
```diff
@@ -32,3 +32,19 @@ asset-hub-kusama-runtime = { path = "../../../../runtimes/assets/asset-hub-kusam
...
+	"frame-system/runtime-benchmarks",
+	"integration-tests-common/runtime-benchmarks",
+	"pallet-asset-conversion/runtime-benchmarks",
+	"pallet-assets/runtime-benchmarks",
+	"pallet-balances/runtime-benchmarks",
+	"pallet-xcm/runtime-benchmarks",
+	"polkadot-parachain-primitives/runtime-benchmarks",
+	"polkadot-runtime-parachains/runtime-benchmarks",
+	"polkadot-runtime/runtime-benchmarks",
+	"sp-runtime/runtime-benchmarks",
```
Comment (NachoPal) [inline cumulus/parachains/integration-tests/emulated/assets/asset-hub-polkadot/Cargo.toml:49]: Is all this necessary? `runtime-benchmarks = []` wouldn't be enough?

**Review by NachoPal — APPROVED** [review]

**Review by bkontur — APPROVED** [review]

**Pull request merged**